src/ci/docker/scripts/fuchsia-test-runner.py

```diff
@@ -276,27 +277,60 @@ def start(self):
             stderr=self.subprocess_output(),
         )
 
-        # Start emulator
-        self.log_info("Starting emulator...")
-        product_bundle = "terminal.qemu-" + self.triple_to_arch(self.target)
+        # Look up the product bundle transfer manifest.
+        self.log_info("Looking up the product bundle transfer manifest...")
+        product_name = "minimal." + self.triple_to_arch(self.target)
+        fuchsia_version = "14.20230811.2.1"
+
+        # FIXME: We should be able to replace this with the machine parsable
+        # `ffx --machine json product lookup ...` once F15 is released.
+        out = subprocess.check_output(
+            [
+                ffx_path,
+                "product",
+                "lookup",
+                product_name,
+                fuchsia_version,
+                "--base-url",
+                "gs://fuchsia/development/" + fuchsia_version,
+            ],
+            env=ffx_env,
+            stderr=self.subprocess_output(),
+        )
+
+        self.log_debug(out)
+
+        for line in io.BytesIO(out):
+            if line.startswith(b"gs://"):
+                transfer_manifest_url = line.rstrip()
+                break
+        else:
+            raise Exception("Unable to parse transfer manifest")
+
+        # Download the product bundle.
+        product_bundle_dir = os.path.join(self.tmp_dir(), 'product-bundle')
```

Comment:
```suggestion
        product_bundle_dir = os.path.join(self.tmp_dir(), "product-bundle")
```

nit since everything else in this file uses double quotes

Reply:
done in latest version.